sanity-studio/schemas/time-range.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  title: 'Time Range',
```

Comment:
'Time range' usually indicates there's a starting time and an ending time, while it's true for a yoga class to have a time range, I think the schema name should still be 'time'

Reply:
It makes sense to have a name as 'time' since it combines either the start time or the end time in the end.
Thank you!

Reply:
'time' is a reserved name in Sanity, so I'll update it with 'Class Time'